HW04/alexey_shparun/lib/student.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Student
+  attr_reader :name, :surname
+  attr_accessor :homeworks_list, :notifications_list
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @notifications_list = []
+    @homeworks_list = []
+  end
+
+  def notify_mentor(notification)
+    notifications_list.push(notification)
+  end
+
+  def mark_as_read!
+    notifications_list.clear
+  end
+
+  def to_work!(homework)
+    notification = Notification.new("#{homework.title} taken to work").display
+    notify_mentor(notification)
+  end
+
+  def add_answer!(homework, answer)
+    homework.answer_list.push(answer)
+  end
+
+  def to_check!(homework)
+    notification = Notification.new("#{homework.title} ready for review").display
+    notify_mentor(notification)
+  end
+
+  def homeworks
+    @homeworks_list.each { |homework| puts "Title: #{homework.title}, Description: #{homework.description}" }
```

Comment:
Let's also not mix business logic with displaying information on the screen. Just return all homework here , and in app.rb you can display them however you want